### DISCUSSION TITLE
MainImage in Body

### DISCUSSION THREAD

**Discussion by adgad** [Financial-Times/content-tree#20] [Team Posts]
The mainImage is added to the top of bodyXML, however it is never used (by Customer Products) in this context - as we always display it in the topper (above the byline)
However, the presence of the mainImage in the body is the only thing that determines whether or not to display the topper mainImage.

### Proposed solution
(as discussed in December in content-pipeline call)

- Keep mainImage in bodyXML for backwards compatibility
- New bodyTree field will not have mainImage in it
- Add a new boolean property somewhere indicating whether or not to show the mainImage

(Arjun note: this solution is fine, however decisions on some of the other image issues might change how we solve this)

**Comment by atanasdinov** [comment]
The proposed solution is fine, however, we'd still need to double-check whether any external clients are using the embedded image or not. This would allow us to safely deprecate `bodyXML`.
